### PR TITLE
cleanup: fix pom xml to add namespace

### PR DESCRIPTION
**Pull request by Shabirmean**
## Description

- This is a cleanup done in preparation to the Q2-23 fixit 
- I am trying to fix all the incorrect GroupId usages in the pom xml to `com.example.product`.
- Manually doing this is impossible. Thus I need to do it programatically.
- For me to be able to programatically be able to parse the xml files it need to have the correct namespaces
- In a followup PR I will open the proper GroupID change

## Checklist
- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: trace/pom.xml [hunk 1]
```diff
@@ -14,7 +14,9 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (Sita04) [inline trace/pom.xml:19]: Apologies for nit-picking. 
Can we intend this in alignment with other files?

File: opencensus/pom.xml [hunk 2]
```diff
@@ -15,7 +15,9 @@
  limitations under the License.
 -->
 <!-- [START opencensus_pom] -->
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (Sita04) [inline opencensus/pom.xml:20]: Can we intend this in alignment with other files?

File: mlengine/online-prediction/pom.xml [hunk 3]
```diff
@@ -11,7 +11,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (Sita04) [inline mlengine/online-prediction/pom.xml:16]: Can we intend this in alignment with other files?

File: dialogflow/basic-webhook/pom.xml [hunk 4]
```diff
@@ -14,7 +14,9 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (Sita04) [inline dialogflow/basic-webhook/pom.xml:19]: Can we intend this in alignment with other files?

**Review by Sita04 — APPROVED** [review]
Thanks Shabir! LGTM

File: mlengine/online-prediction/pom.xml (hunk 3 above)
Comment (Shabirmean) [reply mlengine/online-prediction/pom.xml]: Fixed it!

File: opencensus/pom.xml (hunk 2 above)
Comment (Shabirmean) [reply opencensus/pom.xml]: Fixed it!

File: dialogflow/basic-webhook/pom.xml (hunk 4 above)
Comment (Shabirmean) [reply dialogflow/basic-webhook/pom.xml]: Fixed it!Fixed it!

**Pull request merged**